concepts/eventing-code-guidelines.md

```diff
@@ -389,6 +389,64 @@ MarkReady(apiRuleNew) // instead consider using another function that explicitly
 ```
 </details>
 
+### Logging Guidelines
+
+Here are some general rules to standardize logs across eventing components:
+
+- Log message should be as short and meaningful as possible.
+- Logs should be aligned with the unified way of logging inside kyma ([reference](https://github.com/kyma-project/community/blob/main/concepts/observability-consistent-logging/README.md)).
+- Each log message should have enough context to convey what happened.
+- Each log message should have the proper log level ([reference](https://github.com/kyma-project/community/blob/main/concepts/observability-consistent-logging/unified-approach-to-logging-levels.md)).
+
+Code-specific rules:
+
+- Consider using `uber-go/zap` logging library. Label the logger by naming it with the component name and add the context whenever it is possible ([reference](https://github.com/kyma-project/community/blob/main/concepts/observability-consistent-logging/improvement-of-log-messages-usability.md#log-structure)):
+  <details>
+        <summary>Example</summary>
+
+  ```go
+  namedLogger := r.logger.WithContext().Named("logger-name").With("backend", "BEB")
+  r.namedLogger().Info("Creating Event Publisher deployment")
+  ```
+  will output:
+  ```
+  {"level":"INFO","timestamp":"2022-07-04T12:00:36+02:00","logger":"logger-name","caller":"backend/reconciler.go:741","message":"Creating Event Publisher deployment","context":{"backend":"BEB"}}
+  ```
+  </details>
+
+
+- Don't log the error in case you return the same error as a result for the `Reconcile()` method. **Reason:** kubebuilder will output it too, so in the end the user will have two very similar logs one after another:
```

Comment:
```suggestion
-If you return the same error as a result for the `Reconcile()` method, don't log the error. That's because Kubebuilder will output it too, so the user gets two very similar logs one after another:
```

Reply:
done